### ISSUE TITLE
[BUG] stop function gives error "can't read property url of undefined"

### ISSUE THREAD

**Issue by SpeckyYT** [SushiBtw/discord-music-player#81]
**Bug Description**
As the title suggests, the stop function throws the error "cannot read property url of undefined", which links to `Player.js line 666` 😳
```js
const stream = ytdl(song.url, {
```

**Reproduction Steps**
Code used for playing:
```js
    await bot.music.playlist(msg, searchObj);
    bot.music.toggleQueueLoop(msg);
    bot.music.shuffle(msg);
    bot.music.skip(msg);
```
Code used for stopping:
```js
bot.musc.stop(msg);
```

**Expected Result**
What I expect are no errors.

**Versioning:**
 - OS: Windows 10
 - NPM Version: 6.14.10
- Module Version: 7.0.0

**Additional Comments**
![image](https://user-images.githubusercontent.com/40666126/111033615-63963c00-8412-11eb-90ac-4f32478e8027.png)

**Comment by kaly666** [comment]
The issue occurs in Player.js line 806. "song" seems to be undefined.

**Comment by SushiBtw** [comment]
I will be pushing a ``v7.0.0-fix1`` update for it in 1 hour.

**Comment by SushiBtw** [comment]
I identified the issue that happened when the `repeatMode` was `true`, I am working on a fix - I will keep you updated.

**Comment by SushiBtw** [comment]
Thank you for the patience, I released a `v7.0.0-fix1` update on NPM.
All should be fixed, please check and let me know :)

https://www.npmjs.com/package/discord-music-player

**Comment by SpeckyYT** [comment]
it works correctly now, thanks
